docs/docs/tutorials/simple_hello_world.md

```diff
@@ -0,0 +1,30 @@
+---
+layout: default
+title: Creating a simple hello world with gRPC
+parent: Tutorials
+nav_order: 2
+---
+
+## Creating a simple hello world with gRPC
+
+To understand the gRPC-Gateway we are going to first make a hello world gRPC service.
+
+### Defining your gRPC service using protocol buffers
+
+Before we create a gRPC service, we should create a proto file to define what we need, here we create a file named `hello_world.proto` in the directory `proto/helloworld/hello_world.proto`.
+
+```proto
+syntax = "proto3";
+
+package helloworld;
+
```

Comment:
We are missing the service definition, we need it in this file (without the `google.http.api` annotations):
```suggestion
syntax = "proto3";

package helloworld;

// Here is the overall greeting service definition where we define all our endpoints
service Greeter {
  // Sends a greeting
  rpc SayHello (HelloRequest) returns (HelloReply) {}
}

```